index.html

```diff
@@ -80,6 +80,216 @@ <h2 class="purple">LET'S CONNECT</h2> <br>
+                     <li>CSS</li>
+                     <li>JavaScript</li>
+                 </ul>
+ 
+                 <button class="btn" type="button">See Project</button>
+             </article>
+
+             <article class="card">
+                <img src="images/project3.PNG" alt="project1">
+                <h2>Tonic</h2>
+                <ul>
+                 <li class="no-bullet">CANOPY</li>
+                 <li class="project-tag"> 
+                     <span> <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                     <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                     </svg>
+                      Back End Dev
+                     </span>
+                 </li>
+                 <li class="project-tag">
+                      <span> <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                     <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                     </svg>
+                      2015
+                     </span>
+                 </li>
+                </ul>
+                <p>A daily selection of
+                  privately personalized reads; no accounts or sign-ups required.</p>
+ 
+                 <ul class="tech-stack">
+                     <li>HTML</li>
+                     <li>CSS</li>
+                     <li>JavaScript</li>
+                 </ul>
+ 
+                 <button class="btn" type="button">See Project</button>
+             </article>
+
+             <article class="card">
+                <img src="images/project2.PNG" alt="project1">
+                <h2>Multi Post Stories</h2>
+                <ul>
+                 <li class="no-bullet">CANOPY</li>
+                 <li class="project-tag"> 
+                     <span> <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                     <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                     </svg>
+                      Back End Dev
+                     </span>
+                 </li>
+                 <li class="project-tag">
+                      <span> <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                     <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                     </svg>
+                      2015
+                     </span>
+                 </li>
+                </ul>
+                <p>A daily selection of
+                  privately personalized reads; no accounts or sign-ups required.</p>
+ 
+                 <ul class="tech-stack">
+                     <li>HTML</li>
+                     <li>CSS</li>
+                     <li>JavaScript</li>
+                 </ul>
+ 
+                 <button class="btn" type="button">See Project</button>
+             </article>
+        </section>
+
+        <section class="about">
+            <h1 class="heading">About Myself</h1>
```

Comment:
- [ ] Kindly I recommend you to use a header tag `hx(h2)` tag for the title (e.g., "About myself") instead of` h1` tag for the purpose of search and optimization in your HTML code , it better to have only one h1 header tag in your code .Kindly fix it.